### PR TITLE
Misc Bugfixes

### PR DESCRIPTION
**Pull request by Snakebittenn**
- bugfix: "Unathi forward demon horns now look right."
  - bugfix: "Fingerless gloves work right again."
  - bugfix: "Orange and science goggles are pickable again."

### CODE REVIEW

**Comment by Snakebittenn** [comment]
!review

**Comment by Snakebittenn** [comment]
!bugfix

File: code/modules/clothing/gloves/color.dm [hunk 1]
```diff
@@ -166,10 +166,10 @@
 	name = "fingerless gloves"
 	icon_state = "fingerlessgloves"
 	item_state = "fingerlessgloves"
+	fingerprint_chance = 100
+	clipped = 1
```
Comment (Geevies) [inline code/modules/clothing/gloves/color.dm:170]:
```suggestion
	clipped = TRUE
```

**Review by Geevies — APPROVED** [review]

File: code/modules/clothing/gloves/color.dm (hunk 1 above)
Comment (Snakebittenn) [reply code/modules/clothing/gloves/color.dm]: Handled

**Review by Alberyk — APPROVED** [review]

**Pull request merged**